testdata/src/a/a.go

```diff
@@ -94,4 +94,27 @@ func ok() {
 			Str("bar", "baz").
 			Int("n", 1),
 		).Send()
+
+	// custom object marshaller
+	f := &Foo{Bar: &Bar{}}
+
+	log.Info().Object("foo", f).Msg("")
+}
+
+type Marshaller interface {
+	MarshalZerologObject(event *zerolog.Event)
+}
+
+type Foo struct {
+	Bar Marshaller
+}
```

Comment:
So does it have to be `MarshalZerologObject` nesting `MarshalZerologObject` in it? Not just a simple use of the custom marshaler?

Reply:
The panic occurs when a method of an interface is called with a `zerolog.Event`. The `ssa.CallCommon` value is therefore an `ssa.UnOp` which is not a `StaticCallee`. 

I managed to isolate the issue to its minimal form:

```
type Foo struct {
	Bar zerolog.LogObjectMarshaler
}

func (f *Foo) MarshalZerologObject(event *zerolog.Event) {
	f.Bar.MarshalZerologObject(event)
}
``` 

There are (probably) other ways to reproduce the issue: You need a function or a method call which takes at least one `zerolog.Event` and this call needs to be something other than `ssa.Function` or `ssa.MakeClosure`.

Reply:
Understood. Thank you for elaborating.